mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -191,7 +192,7 @@ def Rock_ReduceOp :
   let summary = "Axes-wide reduction operation";
   let hasVerifier = 1;
   let assemblyFormat = [{
-    $reduceMethod $in `into` $out attr-dict `:` type($in) `into` type($out) (`->` type($result)^)?
+    $reduceMethod $in `into` $out `features` `=` $features attr-dict `:` type($in) `into` type($out) (`->` type($result)^)?
```

Comment:
... hold on, since we're here, can you do me a favor and see what happens if we don't explicitly mention `$features`? I wonder how ugly the syntax will get. That's probably a future PR though